src/redist/targets/BundledTemplates.targets

```diff
@@ -20,6 +20,21 @@
 
   </Target>
 
+  <ItemGroup>
+    <Bundled80Templates Include="Microsoft.DotNet.Common.ItemTemplates" PackageVersion="$(MicrosoftDotNetCommonItemTemplates80PackageVersion)" />
+    <Bundled80Templates Include="Microsoft.DotNet.Common.ProjectTemplates.8.0" PackageVersion="$(MicrosoftDotNetCommonItemTemplates80PackageVersion)" />
```

Comment:
I think the build is currently complaining about this line:
```
Determining projects to restore...
/Users/runner/work/1/s/artifacts/obj/redist/Release/net7.0/TemplatePackageDownloader/TemplatePackageDownloader.csproj : error NU1101: Unable to find package Microsoft.DotNet.Common.ProjectTemplates.8.0. No packages exist with this id in source(s): dotnet-eng, dotnet-libraries, dotnet-public, dotnet-tools, dotnet3-transport, dotnet3.1-transport, dotnet5, dotnet5-transport, dotnet6, dotnet6-transport, dotnet7, dotnet7-transport, dotnet8, dotnet8-transport, xamarin
##[error]artifacts/obj/redist/Release/net7.0/TemplatePackageDownloader/TemplatePackageDownloader.csproj(0,0): error NU1101: (NETCORE_ENGINEERING_TELEMETRY=Build) Unable to find package Microsoft.DotNet.Common.ProjectTemplates.8.0. No packages exist with this id in source(s): dotnet-eng, dotnet-libraries, dotnet-public, dotnet-tools, dotnet3-transport, dotnet3.1-transport, dotnet5, dotnet5-transport, dotnet6, dotnet6-transport, dotnet7, dotnet7-transport, dotnet8, dotnet8-transport, xamarin
  Failed to restore /Users/runner/work/1/s/artifacts/obj/redist/Release/net7.0/TemplatePackageDownloader/TemplatePackageDownloader.csproj (in 24.94 sec).
```

We should probably comment it out too.